### ISSUE TITLE
SPM "new segment" tissues definition

### ISSUE THREAD

**Issue by bclenet** [Inria-Empenn/narps_open_pipelines#232]
When using the new segmentation from SPM in Nipype, we currently write the following lines:

```
# NEW SEGMENT - Unified segmentation using tissue probability maps in SPM12.
spm_tissues_file = join(SPMInfo.getinfo()['path'], 'tpm', 'TPM.nii')
segmentation = Node(NewSegment(), name = 'segmentation')
segmentation.inputs.write_deformation_fields = [False, True]
segmentation.inputs.tissues = [
    [(spm_tissues_file, 1), 2, (True, False), (True, False)], # Grey matter
    [(spm_tissues_file, 2), 2, (True, False), (True, False)], # White matter
    [(spm_tissues_file, 3), 2, (True, False), (True, False)], # CSF
    [(spm_tissues_file, 4), 3, (True, False), (True, False)], # Bone
    [(spm_tissues_file, 5), 4, (True, False), (True, False)], # Soft tissue
    [(spm_tissues_file, 6), 2, (True, False), (True, False)] # Air / background
]
```
where:

* `spm_tissues_file` is the tissue probability map file [`TPM.nii`](https://github.com/spm/spm12/blob/main/tpm/TPM.nii) of SPM
* each element of the `segmentation.inputs.tissues` represents a probability map ; the second element of each being the *Number of Gaussians* (cf. [SPM manual](https://www.fil.ion.ucl.ac.uk/spm/doc/spm12_manual.pdf) page 49) for each tissue.

**Currently, I'm not able to find if there are default values for the number of gaussians. And if not, which ones we should set.**

The manual (page 50) mentions:
> Typical numbers of Gaussians could be two for grey matter, two for white matter, two for CSF, three for bone, four for other soft tissues and two for air (background).

**Comment by bclenet** [comment]
@cmaumet, could you please let me know if you have an idea about that ?
Thanks !

**Comment by bclenet** [comment]
For now, this is needed by:

- 98BT
- 0I4U
- V55J

**Comment by cmaumet** [comment]
@bclenet: You are right! The signature of [spm12/spm_preproc.m](https://github.com/spm/spm12/blob/main/spm_preproc.m) looks very close to [spm5/spm_preproc.m](https://github.com/spm/spm5/blob/main/spm_preproc.m) so this must be the 'old segment' and then [spm12/spm_preproc8.m](https://github.com/spm/spm12/blob/main/spm_preproc8.m) is the 'new segment'. I was not able to locate the default value from the code itself but if we launch spm this should be easy to find. I can't right now because there is no license left but let's look at this together next time we meet.

**Comment by cmaumet** [comment]
Hi @bclenet!

I've managed to open SPM, here are the defaults:

![Image](https://github.com/user-attachments/assets/f914d406-cd1e-454a-95d6-a5c145aba6c0)

Does this help?

**Comment by bclenet** [comment]
Hi @cmaumet,
It looks like the information we needed. Thanks !
